### PR TITLE
wsr-types: add types to InputWithLabel

### PR DESCRIPTION
**Pull request by itaytay**

### CODE REVIEW

File: packages/wsr-types/src/drivers/InputWithLabel.uni.driver.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+declare namespace __WSRTests {
```
Comment (mosheKerbel) [inline packages/wsr-types/src/drivers/InputWithLabel.uni.driver.d.ts]: modify

File: packages/wsr-types/src/components/InputWithLabel.d.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+declare namespace __WSR {
+  namespace InputWithLabel {
+    export interface InputWithLabelProps {
+      dataHook?: string;
+      suffix?: React.ReactNode;
+      label?: string;
+      value?: string | number;
+      status?: InputWithLabelStatus;
+      statusMessage?: React.ReactNode;
+      onFocus?: (e?: FocusEvent) => void;
```
Comment (mosheKerbel) [inline packages/wsr-types/src/components/InputWithLabel.d.ts]: check if we can use the given react type

File: packages/wsr-types/src/drivers/InputWithLabel.uni.driver.d.ts (hunk 1 above)
Comment (itaytay) [reply packages/wsr-types/src/drivers/InputWithLabel.uni.driver.d.ts]: done

File: packages/wsr-types/src/components/InputWithLabel.d.ts (hunk 2 above)
Comment (itaytay) [reply packages/wsr-types/src/components/InputWithLabel.d.ts]: done

File: packages/wsr-types/src/testkits/vanilla.d.ts [hunk 3]
```diff
@@ -68,6 +68,8 @@ declare module "wix-style-react/dist/testkit" {
     HTMLElement
   >>;
   export const inputWithOptionsTestkitFactory: VanillaTestkitFactory<__WSRTests.InputWithOptionsDriver>;
+  export const inputWithLabelTestkitFactory: VanillaUniTestkitFactory<__WSRTests.InputWithLabelUniDriver>;
+  export const autoCompleteWithLabelTestkitFactory: VanillaUniTestkitFactory<__WSRTests.InputWithLabelUniDriver>;
```
Comment (mosheKerbel) [inline packages/wsr-types/src/testkits/vanilla.d.ts]: __WSRTests.AutoCompleteWithLabelUniDriver

File: packages/wsr-types/src/testkits/puppeteer.d.ts [hunk 4]
```diff
@@ -42,6 +42,8 @@ declare module "wix-style-react/dist/testkit/puppeteer" {
   export const linearProgressBarTestkitFactory: PuppeteerUniTestkitFactory<__WSRTests.LinearProgressBarUniDriver>;
   export const errorIndicatorTestkitFactory: PuppeteerUniTestkitFactory<__WSRTests.ErrorIndicatorUniDriver>;
   export const addItemTestkitFactory: PuppeteerUniTestkitFactory<__WSRTests.AddItemUniDriver>;
+  export const inputWithLabelTestkitFactory: PuppeteerUniTestkitFactory<__WSRTests.InputWithLabelUniDriver>;
+  export const autoCompleteWithLabelTestkitFactory: PuppeteerUniTestkitFactory<__WSRTests.InputWithLabelUniDriver>;
```
Comment (mosheKerbel) [inline packages/wsr-types/src/testkits/puppeteer.d.ts]: __WSRTests.InputWithLabelUniDriver

File: packages/wsr-types/src/testkits/enzyme.d.ts [hunk 5]
```diff
@@ -69,6 +69,8 @@ declare module "wix-style-react/dist/testkit/enzyme" {
     ReactWrapper
   >>;
   export const inputWithOptionsTestkitFactory: EnzymeTestkitFactory<__WSRTests.InputWithOptionsDriver>;
+  export const inputWithLabelTestkitFactory: EnzymeUniTestkitFactory<__WSRTests.InputWithLabelUniDriver>;
+  export const autoCompleteWithLabelTestkitFactory: EnzymeUniTestkitFactory<__WSRTests.InputWithLabelUniDriver>;
```
Comment (mosheKerbel) [inline packages/wsr-types/src/testkits/enzyme.d.ts]: __WSRTests.InputWithLabelUniDriver

File: packages/wsr-types/src/components/InputWithLabel.d.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
+declare namespace __WSR {
+  namespace InputWithLabel {
+    export interface InputWithLabelProps {
+      dataHook?: string;
+      suffix?: React.ReactNode[];
+      label?: string;
+      value?: string | number;
+      status?: InputWithLabelStatus;
```
Comment (mosheKerbel) [inline packages/wsr-types/src/components/InputWithLabel.d.ts]: missing 'error' and 'loading'

File: packages/wsr-types/src/components/AutoCompleteWithLabel.d.ts [hunk 7]
```diff
@@ -0,0 +1,72 @@
+declare namespace __WSR {
+  namespace AutoCompleteWithLabel {
+    export interface AutoCompleteWithLabelProps {
+      dataHook?: string;
+      label: string;
+      suffix?: React.ReactNode[];
+      options: DropdownLayoutOption[];
+      status?: AutoCompleteWithLabelStatus;
```
Comment (mosheKerbel) [inline packages/wsr-types/src/components/AutoCompleteWithLabel.d.ts]: missing 'error' and 'loading'

**Comment by mosheKerbel** [comment]
@itaytay I'm fixing it, please do not update PR

File: packages/wsr-types/src/components/InputWithLabel.d.ts (hunk 6 above)
Comment (itaytay) [reply packages/wsr-types/src/components/InputWithLabel.d.ts]: That's how I so this exact use case implemented in Input and in NoBorderInput.
Should I still do it differently?

File: packages/wsr-types/src/components/AutoCompleteWithLabel.d.ts (hunk 7 above)
Comment (itaytay) [reply packages/wsr-types/src/components/AutoCompleteWithLabel.d.ts]: Same answer as above

**Pull request merged**